index.html

```diff
@@ -0,0 +1,204 @@
+        </li>
+        <li class="me-4">
+          <a href="#">
+            <i class="fab fa-twitter"></i>
+          </a>
+        </li>
+        <li class="me-4">
+          <a href="#">
+            English
+          </a>
+        </li>
+        <li class="me-4">
+          <a href="#">
+            My Page
+          </a>
+        </li>
+        <li class="me-70">
+          <a href="#">
+            Log Out
+          </a>
+        </li>
+      </ul>
+      <nav class="navbar navbar-expand-lg navbar-light bg-light pt-0 justify-content-evenly pt-2">
+        <div class="container-fluid">
+          <a class="navbar-brand d-lg-block d-sm-none d-none ms-5" href="/">
+            <img src="./imgs/logo1.jpg" alt="logo" class="img-fluid"/>
+          </a>
+          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+            <span class="navbar-toggler-icon"></span>
+          </button>
+          <div class="collapse navbar-collapse w-100" id="navbarSupportedContent">
+            <ul class="navbar-nav mb-2 mb-lg-0 align-items-center w-100 justify-content-lg-end justify-content-sm-evenly justify-content-evenly me-lg-5">
+              <li class="nav-item d-lg-none d-sm-block d-block">
+                <a class="nav-link navbar-brand w-100" href="/">
+                  <img src="./imgs/logo.jpg" alt="logo" class="img-fluid"/>
+                </a>
+              </li>
+              <li class="nav-item me-lg-5">
+                <a class="nav-link active" aria-current="page" href="#">Home</a>
+              </li>
+              <li class="nav-item me-lg-5">
+                <a class="nav-link" href="./pages/about.html">About</a>
+              </li>
+              <li class="nav-item me-lg-5">
+                <a class="nav-link" href="#">Program</a>
+              </li>
+              <li class="nav-item me-lg-5">
+                <a class="nav-link" href="#">Join</a>
+              </li>
+              <li class="nav-item me-lg-5">
+                <a class="nav-link" href="#">contact</a>
+              </li>
+              <li class="nav-item me-lg-5 p-1 border border-success">
+                <a class="nav-link text-success" href="#">
+                  Log Out
+                </a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </nav>
+    </header>
+    <!-- Start Headline -->
+    <section class="headline container mt-3">
+      <h1 class="d-block mb-2">
+        "Hello Future Programmers"
+      </h1>
+      <img src="./imgs/headline.png" alt="Creatve Programmers and UI UX Designers" class="col-lg-9 col-sm-10 col-12"/>
+      <p class="ms-md-5 ms-3">
+        if you want to be a professional web developer. 
+        So it's time to take our course and become a professional developer. 
+        We are a group of professional teachers.
```

Comment:
- [ ] You need to add a border to this paragraph like the original design. And you missed the date of the event. Please, fix it.

Your design:

![imagen](https://user-images.githubusercontent.com/60631456/132544123-2f974bb0-9b00-4c67-85b1-6202e7226c53.png)

Original design:

![imagen](https://user-images.githubusercontent.com/60631456/132544233-62815c1b-b4a6-434d-937b-bbeac5fff37f.png)

Reply:
- Kindly add a border to this paragraph like the original design. And you missed the date of the event.

**- Your design:**

![](https://user-images.githubusercontent.com/60631456/132544123-2f974bb0-9b00-4c67-85b1-6202e7226c53.png)

**- Requested design:**

![](https://user-images.githubusercontent.com/60631456/132544233-62815c1b-b4a6-434d-937b-bbeac5fff37f.png)

Reply:
Worked on this!!